packages/zarf-injector/zarf.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
 
 components:
   - name: zarf-injector
+    description: |
+      Injects Zarf into the cluster so it can bootstrap other core Zarf components. 
```

Comment:
I think it might help to expand on this instead of saying it "injects Zarf".  There's bound to be some confusion about the Zarf Injector and the mutating webhook.  Perhaps having a reference back to a page in the docs, or something with more information about what the injector is, or using the injector name so someone using it could find it more easily in the docs would be helpful

Reply:
perhaps `A tiny Rust binary that is used to setup the in-cluster seed registry when an external registry is unavailable.  See https://github.com/defenseunicorns/zarf/blob/main/adr/0007-use-rust-binary-for-both-injection-stages.md for more information.`

Reply:
I'd probably word it something like bootstraps a cluster with a single binary to load the Zarf Registry. Is probably more than sufficient for this. It's not like they can skip deploying it, if it's needed it will be deployed regardless.

Reply:
> I'd probably word it something like bootstraps a cluster with a single binary to load the Zarf Registry. Is probably more than sufficient for this. It's not like they can skip deploying it, if it's needed it will be deployed regardless.

`Bootstraps a cluster using a tiny Rust binary to deploy the internal Zarf Registry`?

Reply:
My thoughts: What would make the most sense to someone like Ashton? I think the above language is concise and clearly describes functionally. 

@runyontr I would be hesitant to include links to the docs since airgap users may not have access to those at this time.

Reply:
Agreed.  Perhaps the docs should be added to the ZarfUI for airgap help?

Reply:
@runyontr It is something the team has discussed - there may be an issue for it if you would like to add your support. :) But I would want to heavily consider who would be using the docs in Airgap too. Right now the docs are written for a number of different users - not just Ashton